services/core/shared/core/compat/sdk_v5/transactions.js

```diff
@@ -208,7 +208,19 @@ const getTransactions = async params => {
 	return transactions;
 };
 
+const getTransactionsByBlockId = async blockId => {
+	const transactions = await getTransactions({ blockId });
+	return {
+		data: transactions,
+		meta: {
+			offset: 0,
+			count: transactions.length,
+		},
+	};
+};
+
```

Comment:
`getTransactionsByBlockId` can be simply reduced to:
The standard response envelope (Line 174, `const transactions`) is already defined inside `getTransactions`.

```suggestion
const getTransactionsByBlockId = async blockId =>  getTransactions({ blockId });

```

Reply:
Done!